roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,22 +15,37 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
         },
+        "online": {
```

Comment:
@rhcarvalho 
> Would be good to investigate how is it used and what is the intention of setting deployment_type=online in the inventory file.

After speaking with @sallyom, she clarified that "online" is an old type that was valid until release 1.3 of openshift-ansible. When using "online", `openshift3/ose-*` images were used as well. I have updated the PR to address this. 

We could remove `online` altogether from this PR and just treat it as an invalid deployment type if you think this would be the better approach.